.gitignore

```diff
@@ -10,6 +10,9 @@ tmtags
 \#*
 .\#*
 
+## RUBYMINE
```

Comment:
.idea directories are used by all Jetbrains IDEs.  I added it to the ignores to be on par with the VIM ignores on line 16.